# Message used to define a bounding region by its min/max x,y z values.
float64 x_min
float64 x_max
float64 y_min
float64 y_max
float64 z_min
float64 z_max